src/components/FormComponents/FormComponents.js

```diff
@@ -214,7 +225,14 @@ export const PhoneNumber = ({ slug, required, bindField, collectAnswer }) => {
                 required={required}
                 className="TextInput"
                 {...bindField(slug)}
-                onChange={(e) => collectAnswer(slug, e.target.value)}
+                onChange={(e) => {
+                    if (e.target.checkValidity()) {
```

Comment:
what is this checking?

Reply:
It's checking if what the user inputs is in the correct format then sets the "errors" state. If there are errors it won't submit the form.

We are asking for a specific format for the phone number, email, and zip code.